docs/tabs.md

```diff
@@ -0,0 +1,71 @@
+# Tabs
+### Usage
+
+```jsx
+import { Tabs } from 'radiance-ui';
+
+const Content = styled.div`
+  ${TYPOGRAPHY_STYLE.display};
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  background-color: ${props => props.bgColor};
+  color: ${COLORS.primary};
+  height: calc(100vh / 2);
+`;
+
+class TabsExample extends React.Component {
+  state = {
+    activeTabId: 1,
+  };
+
+  tabItems = [
+    { id: 1, text: 'Tab 1', bgColor: COLORS.postcardGreen },
+    { id: 2, text: 'Tab 2', bgColor: COLORS.postcardPeach },
+    { id: 3, text: 'Tab 3', bgColor: COLORS.postcardYellow },
+  ];
+
+  onClickTab = tab => {
+    this.setState({ activeTabId: tab.id });
```

Comment:
it looks like from the implementation, the `<Tabs />` component handles the state for which tab is currently active. Having this in the example is a bit misleading. Maybe we can just console.log out the `tab` variable and remove having `activeTabId` as state on this component

Reply:
done